roman.py

```diff
@@ -33,21 +33,21 @@ def get_roman_digit(number, base):
     if quotient >= 5:
         # do something including the half base
         if quotient == 9:
-            sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*10)])) 
+            sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*10)])) 
         else:
-            sys.stdout.write('%s' % roman_digit[(base*5)])
+            sys.stdout.write('%s' % ROMAN_DIGIT[(base*5)])
             remaining = quotient - 5
-            sys.stdout.write('%s' % roman_digit[base] * remaining)
+            sys.stdout.write('%s' % ROMAN_DIGIT[base] * remaining)
```

Comment:
i have a pretty strong aversion to the C-stye string formatting, preferring the curly brace / `str.format` thing. https://docs.python.org/2/library/string.html#format-examples

up to you tho.

(fun aside: python 3.6 introduced interpolation https://www.python.org/dev/peps/pep-0498/ , something I would assume your perl mind would appreciate)